src/sentry/workflow_engine/endpoints/validators/base/detector.py

```diff
@@ -23,8 +23,14 @@ class BaseDetectorTypeValidator(CamelSnakeSerializer):
     name = serializers.CharField(
         required=True,
         max_length=200,
-        help_text="Name of the uptime monitor",
+        help_text="Name of the detector",
     )
+    # description = serializers.CharField(
+    #     required=False,
+    #     help_text="Description of the detector"
+    # )
+    # owner_user_id = serializers.IntegerField(required=False)
+    # owner_team_id = serializers.IntegerField(required=False)
```

Comment:
kept hitting annoying issues here where even though I set it to optional it was yelling it if wasn't passed, haven't had time to get back to this, it's probably something simple